x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field_helpers.tsx

```diff
@@ -38,6 +45,50 @@ interface RenderRuleNameProps {
   value: string | number | null | undefined;
 }
 
+interface NotDraggableProps {
+  truncate?: boolean;
+  isDraggingDisabled: boolean;
+  fieldName: string;
+  value: string | number | null | undefined;
+  isRoleSeparator?: boolean;
+}
+
+export const NotDraggable: React.FC<NotDraggableProps> = ({
```

Comment:
It may be clearer if we get rid of the negation in the name; is it meant to look like a draggable field but not be interactive? Is it meant to be a sibling of a draggable component that's not interactive and visually distinct?

Reply:
removed